virtool/startup.py

```diff
@@ -365,17 +368,17 @@ async def init_tasks(app: aiohttp.web.Application):
     for subtraction in subtractions_without_fasta:
         await virtool.tasks.pg.register(
             pg,
-            app["task_runner"],
+            app["tasks"],
             WriteSubtractionFASTATask,
             context={"subtraction": subtraction})
 
     logger.info("Checking index JSON files")
 
-    await virtool.tasks.pg.register(pg, app["task_runner"], CreateIndexJSONTask)
-    await virtool.tasks.pg.register(pg, app["task_runner"], DeleteReferenceTask, context={"user_id": "virtool"})
-    await virtool.tasks.pg.register(pg, app["task_runner"], AddSubtractionFilesTask)
-    await virtool.tasks.pg.register(pg, app["task_runner"], StoreNuvsFilesTask)
-    await virtool.tasks.pg.register(pg, app["task_runner"], StoreNuvsFilesTask)
-    await virtool.tasks.pg.register(pg, app["task_runner"], CompressSamplesTask)
+    await virtool.tasks.pg.register(pg, app["tasks"], CreateIndexJSONTask)
+    await virtool.tasks.pg.register(pg, app["tasks"], DeleteReferenceTask, context={"user_id": "virtool"})
+    await virtool.tasks.pg.register(pg, app["tasks"], AddSubtractionFilesTask)
+    await virtool.tasks.pg.register(pg, app["tasks"], StoreNuvsFilesTask)
+    await virtool.tasks.pg.register(pg, app["tasks"], StoreNuvsFilesTask)
+    await virtool.tasks.pg.register(pg, app["tasks"], CompressSamplesTask)
```

Comment:
Get rid of `register`.

Initialize `TaskClient` with `pg` so that API usage can look like:
```py
await app["tasks"].add(CreateIndexJSONTask)
await app["tasks"].add(DeleteReferenceTask, context={"user_id": "virtool"})
await app["tasks"].add(AddSubtractionFilesTask)
```